tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2394 @@
+        }
+    }
+}
+@media print {
+.page {
+width: 8in;
+
+@at-root (with: media) {
+color: red;
+}
+}
+}
+@media print{
+    .page{
+        width: 8in;
+
+        @at-root (with:media){
+            color: red;
+        }
+    }
+}
+@media print {
+    .page {
+        width: 8in;
+
+        @at-root ( with : media ) {
+            color: red;
+        }
+    }
+}
+@media print  {
+    .page  {
+        width: 8in;
+
+        @at-root  (  with  :  media  ) {
+            color: red;
+        }
+    }
+}
+@media print  {
+    .page  {
+        width: 8in;
+
+    @at-root
+        (with: media) {
+            color: red;
+        }
+    }
+}
+@media print
+{
+.page
+{
+width
+:
+8in
+;
+@at-root
+(
+with
+:
+media
+)
+{
+color
+:
+red
+;
+}
+}
+}
+@media print
+
+{
+
+.page
+
+{
+
+width
+
+:
+
+8in
+
+;
+
+@at-root
+
+(
+
+with
+
+:
+
+media
+
+)
+
+{
+
+color
+:
+
+red
+
+;
+
+}
+
+}
+
+}
+@media print {
+    .page {
+        width: 8in;
+
+        @at-root (without: media) {
+            color: red;
+        }
+    }
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root {
+    .child1 {
+      width: 100px;
+    }
+    .child2 {
+      width: 200px;
+    }
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root .child {
+    width: 100px;
+  }
+}
+.parent {
+  @at-root input[type="radio"] {
+    color: red;
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+    @at-root (with: media) {
```

Comment:
Would it make sense to always enforce a bank line before `@at-root`?

Reply:
@j-f1 i think add blank line is bad practice, i am working in many project where people don't add newlines before at rules and always add newline before at rules. I think better output as is. If many contributors vote for the newline can add in the future.